### DISCUSSION TITLE
enlarge emotes on hover

### DISCUSSION THREAD

**Discussion by ovojake** [night/betterttv#6788] [Ideas]
I prefer using bttv but there's still a few features from 7tv that I miss. one of them being the ability to enlarge emotes on hover.

**Comment by exatedlord** [comment]
This!! The enlarging feature that 7TV has is the only feature it has over BTTV, imo. This is a must.

**Comment by night** [comment]
added via 687c0c3ea926b62358726a200f3520b12a00050c

**Reply by ovojake** [reply]
how do I go about adding that? Or is the extension just gonna be updated

**Reply by night** [reply]
it will be in the next update you receive

**Comment by exatedlord** [comment]
So this will be added? That’s huge!

I have some more recommendations that would make BTTV better than 7TV cause
they suck. I’m not sure if it’s a BTTV thing but there is no copy button
next to the reply on certain chat messages when you hover over them. I’ve
attached screenshots for more context.

I will be on the search for other stuff that can be added, I am 100%
dedicated to improving BTTV as much as possible for all users.
- Ted Lord

On Jun 10, 2024, at 4:04 PM, Zack ***@***.***> wrote:

﻿

it will be in the next update you receive

—
Reply to this email directly, view it on GitHub
<https://github.com/night/betterttv/discussions/6788#discussioncomment-9731324>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AMNUMNIBBHOMDDLGZTEZFQDZGYBEZAVCNFSM6AAAAABHHNJG56VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOMZRGMZDI>
.
You are receiving this because you commented.Message ID:
***@***.***>

**Comment by exatedlord** [comment]
Long emote names cut off

- Ted Lord
*770-823-5162 *
